### PR TITLE
Fix AWS provider logging

### PR DESCRIPTION
**Pull request by ArangoGutierrez**

### CODE REVIEW

File: pkg/provider/aws/delete.go [hunk 1]
```diff
@@ -79,8 +79,12 @@ func (a *Client) delete(cache *AWS) error {
 			a.fail()
 			return fmt.Errorf("error deleting security group: %v", err)
 		}
+
+		a.done()
 	}
 
+	a.log.Wg.Add(1)
+	go a.log.Loading("Deleting VPC resources\n")
```
Comment (elezar) [inline pkg/provider/aws/delete.go:87]: Do we need an `\n` here?

**Review by elezar — COMMENTED** [review]
Minor question.

**Review by elezar — APPROVED** [review]
It seems as if the commits can be squashed. Not blockers though.

**Comment by ArangoGutierrez** [comment]
> It seems as if the commits can be squashed. Not blockers though.

Squashed!

**Pull request merged**